### ISSUE TITLE
Looks like twitch updated their api?

### ISSUE THREAD

**Issue by marioandluigi878** [paul-io/momo-2#12]
[momolog.txt](https://github.com/paul-io/momo-2/files/1709310/momolog.txt)
^ entire log is in there but uhh rip?

**Comment by paul-io** [comment]
I did a quick lookthrough and it looks like this relates to #7, where if you don't have a reddit API key set the dashboard breaks. Were you trying to access the dashboard?

**Comment by marioandluigi878** [comment]
look at the very last line to replicate this try to add a twitch channel to the notifications in the dashboard

**Comment by marioandluigi878** [comment]
just basically pops up with a 404 error

**Comment by paul-io** [comment]
Yeah that's just the internal page, I'll get around to fixing #7 really soon here because that's the issue. Temporarily, you could setup the reddit API keys and it should fix the dashboard though
